client/layout/sidebar/footer.jsx

```diff
@@ -15,8 +15,11 @@ import HappychatButton from 'components/happychat/button';
 const SidebarFooter = ( { translate, children } ) => (
 	<div className="sidebar__footer">
 		{ children }
-		<Button compact borderless href="/help">
-			<Gridicon icon="help-outline" /> { translate( 'Help' ) }
+		<Button borderless href={ config( 'signup_url' ) + '?ref=calypso-selector' }>
```

Comment:
Ah! Understood. That functionality should still technically be there:

```
	<div className="sidebar__footer">
		{ children }
```

Incidentally removing the hard-coded Add New Site button from the footer fixes Lance's initial issue.